### PR TITLE
Add tags/branch to git reference quickpick

### LINKED ISSUE #710: Add tags + branch list for a git repo when selecting the ref within QuickPick item

### PR DESCRIPTION
**Pull request by mohitsuman**
Fix #710

### CODE REVIEW

File: src/util/refs.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use strict';
```
Comment (dgolovin) [inline src/util/refs.ts]: There should be original copyright header and I believe we should put link to original repository.

File: src/openshift/component.ts [hunk 2]
```diff
@@ -261,13 +261,8 @@ export class Component extends OpenShiftItem {
...
 
-        const gitRef = await vscode.window.showInputBox({
-            value: 'master',
-            prompt: 'Specify git reference (branch/tags/commits | default: master)',
-            validateInput: (value: string) => {
-                if (!value.trim()) return 'Empty reference, specify master if no reference needed.';
-            }
-        });
+        const references = await Refs.fetchTag(repoURI);
+        const gitRef = await vscode.window.showQuickPick([ ...references.keys()], {placeHolder: "Specify git reference (branch/tags)"});
```
Comment (dgolovin) [inline src/openshift/component.ts]: Should be `Select git reference (branch/tag)`.

File: src/util/refs.ts (hunk 1 above)
Comment (mohitsuman) [reply src/util/refs.ts]: Updated.

File: src/openshift/component.ts (hunk 2 above)
Comment (mohitsuman) [reply src/openshift/component.ts]: Fixed.

**Pull request merged**